master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -1095,22 +1101,35 @@ private[celeborn] class Master(
     }
   }
 
+  private def gaugeShuffleFallbackCounts(): Unit = {
+    statusSystem.shuffleFallbackCounts.entrySet().asScala.foreach { shuffleFallbackCount =>
+      masterSource.addGauge(
+        MasterSource.SHUFFLE_FALLBACK_COUNT,
+        Map("fallbackPolicy" -> shuffleFallbackCount.getKey)) { () =>
+        shuffleFallbackCount.getValue
```

Comment:
I think it is better to get the entry by get during runtime, in case the entry is cleaned, not sure.

```

Option(statusSystem.shuffleFallbackCounts.get(shuffleFallbackCount.getKey)).getOrElse(0)

```